tgbot/handlers/utils/decorators.py

```diff
@@ -0,0 +1,43 @@
+from functools import wraps
+from typing import Callable
+
+from telegram import Update, ChatAction
+from telegram.ext import CallbackContext
+
+from users.models import User
+
+
+def admin_only(reply_message: str = None):
+    """
+    Admin only decorator
+    Used for handlers that only admins have access to
+
+    @param reply_message: message to reply
+    """
+
+    def decorator(func: Callable):
+        @wraps(func)
+        def inner(update: Update, context: CallbackContext, *args, **kwargs):
+            user = User.get_user(update, context)
+
+            if not user.is_admin:
+                if reply_message is not None and update.effective_message:
+                    update.effective_message.reply_text(reply_message)
+                return
```

Comment:
There is unnecessary nesting level with if conditions, it might be reduced.

Reply:
Should I remove 'reply_message' option from the decorator?

Reply:
I think yes, in fact, it's not used because if user is not an admin, bot won't send anything.

Reply:
Yeah I agree with you. I just added it because it was existed before.